style.css

```diff
@@ -40,4 +40,125 @@ button {
 
 .textError {
     color: red;
+}
+
+.repo-card svg {
+    width: 15px;
+}
+
+a {
+    text-decoration: none;
+}
+
+/* Repo-card classes */
+
+.repo-card {
+    border: 1px solid var(--card-border);
+    border-radius: 0.3em;
+    background-color: transparent;
+    color: var(--card-text);
+    padding: 1em;
+}
+
+.repo-link {
+    color: var(--card-text);
+    font-size: 1.1em;
+}
+
+.repo-card a:hover {
+    text-decoration: underline;
+}
+
+.repo-details-style {
+    font-size: 0.8em;
+}
+
+.repo-list {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    list-style: none;
+    padding: 0;
+}
+
+/* Languages */
+
+.lng {
+    position: relative;
+    top: 1px;
+    display: inline-block;
+    width: 11px;
+    height: 11px;
+    border-radius: 50%;
+}
+
+.lng-JavaScript {
+    background-color: var(--js);
+}
+
+.lng-HTML {
+    background-color: var(--html);
+}
+
+.lng-CSS {
+    background-color: var(--css);
+}
+
+.lng-TypeScript {
+    background-color: var(--typescript);
+}
+
+/* Utils */
+
+/* Margins */
+
+.mt-1 {
+    margin-top: 0.5em;
+}
+
+.m-1 {
+    margin: 0.5em;
+}
+
+.ml-2 {
+    margin-left: 1.2em;
+}
+
+.m-3 {
+    margin: 1.5em;
+}
+
+/* Display */
+
+.flex {
+    display: flex;
+}
+
+.span-1 {
+    grid-column: span 1;
+}
+
+.align-center {
+    align-items: center;
+}
+
+.justify-center {
+    justify-content: center;
+}
+
+.justify-around {
+    justify-content: space-around;
+}
+
+/* making the page responsive */
+
+@media screen and (max-width:700px) {
+    .repo-list {
+        grid-template-columns: 1fr 1fr;
+    }
+}
+
+@media screen and (max-width:500px) {
+    .repo-list {
+        grid-template-columns: 1fr;
+    }
```

Comment:
Grid has some special properties that let you make it responsive, without specifying media queries.

Have a look here: https://css-tricks.com/auto-sizing-columns-css-grid-auto-fill-vs-auto-fit/